### PR TITLE
Login dom

### PR DESCRIPTION
**Pull request by ahmedalmi95**
#15

### CODE REVIEW

File: public/js/login.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    };
+    fetch('/login', {
+      method: 'POST',
+      credentials: 'same-origin',
+      body: JSON.stringify(userInfo),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    })
+      .then(res => res.json())
```
Comment (tomduggan85) [inline public/js/login.js:25]: @AhmedAl-Almi here, you could redirect to the next page

Comment (ahmedalmi95) [reply public/js/login.js:25]: done

Comment (tomduggan85) [reply public/js/login.js:25]: Thanks!

**Review by Fatmasiam — APPROVED** [review]

**Review by Angham116 — APPROVED** [review]

**Review by Angham116 — APPROVED** [review]

**Review by Anies12 — APPROVED** [review]

File: public/js/login.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+const email = document.getElementById('email');
+const password = document.getElementById('password');
+const btnLogin = document.getElementById('btn_login');
+const msg = document.getElementById('msg');
```
Comment (kannanhassouna) [inline public/js/login.js]: use descriptive variable name like 
```suggestion
const errorLabel = document.getElementById('msg');
OR
const errorMsg = document.getElementById('msg');

const valiadationLabel = document.getElementById('msg');
```

File: public/js/login.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+const email = document.getElementById('email');
+const password = document.getElementById('password');
+const btnLogin = document.getElementById('btn_login');
```
Comment (kannanhassouna) [inline public/js/login.js]:
```suggestion
const loginBtn = document.getElementById('btn_login');
```

File: public/js/login.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+const msg = document.getElementById('msg');
+
+btnLogin.addEventListener('click', (e) => {
+  e.preventDefault();
+  const emailV = email.value.trim();
+  const passwordV = password.value;
```
Comment (kannanhassouna) [inline public/js/login.js]: why you didn't add trim to password value
```suggestion
  const passwordV = password.value.trim();
  
```

File: public/js/login.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
+const email = document.getElementById('email');
+const password = document.getElementById('password');
+const btnLogin = document.getElementById('btn_login');
+const msg = document.getElementById('msg');
+
+btnLogin.addEventListener('click', (e) => {
+  e.preventDefault();
+  const emailV = email.value.trim();
```
Comment (kannanhassouna) [inline public/js/login.js]: more descriptive names : 
```suggestion
  const emailValue = email.value.trim();
```

**Review by kannanhassouna — CHANGES_REQUESTED** [review]
where the validation  ?

File: public/js/login.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    .then(res => res.json())
+    .then((res) => {
+      if (res.error) {
+        msg.textContent = res.error;
+      } else {
+        window.location.href = '/';
+      }
+    })
+    .catch((err) => {
+      msg.textContent = 'error in request';
```
Comment (kannanhassouna) [inline public/js/login.js]: do you think this message is useful to the user?

File: src/views/login.hbs [hunk 7]
```diff
@@ -0,0 +1,13 @@
+<form id="loginForm" class="loginForm">
```
Comment (kannanhassouna) [inline src/views/login.hbs:1]: remove everything you didn't use it here like : classes

File: src/views/login.hbs [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+  <h1>Login</h1>
+  <div class="emailDiv">
```
Comment (kannanhassouna) [inline src/views/login.hbs]: really,  please add more descriptive names like ``` email-section ```, ``` email-field```

File: src/views/login.hbs [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+    <label for="email">Email</label>
+    <input id="email" type="email" name="email" placeholder="Enter your email" required>
+  </div>
+  <div class="passwordDiv">
+    <label for="password">password</label>
+    <input id="password" type="password" name="password" placeholder="Enter your password" required>
+  </div>
+  <div id="msg"></div>
```
Comment (kannanhassouna) [inline src/views/login.hbs:11]: i think ``` div ``` is not for typograph stuff   
```suggestion
  <span id="msg"></span>
```

Comment (kannanhassouna) [reply src/views/login.hbs:11]: but btw awesome work

File: public/js/login.js [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    })
+    .then(res => res.json())
+    .then((res) => {
+      if (res.error) {
+        msg.textContent = res.error;
+      } else {
+        window.location.href = '/';
```
Comment (kannanhassouna) [inline public/js/login.js]: you should make a function to check validation of the data and test it 

### TESTS

**Review by kannanhassouna — CHANGES_REQUESTED** [review]

**Pull request merged**